packages/smart-vaults/src/vault/SmartVault.sol

```diff
@@ -35,13 +35,35 @@ contract SmartVault is IAccount, Ownable, UUPSUpgradeable, MultiSignerAuth, ERC1
         ERC1271
     }
 
+    /// @notice Upper limits for maxPriorityFeePerGas, preVerificationGas, verificationGasLimit, callGasLimit,
+    /// paymasterVerificationGasLimit and paymasterPostOpGasLimit that should be charged by the userOp. This is included
+    /// in the light userOp hash to ensure last signer does not exceed the specified gas price/limits. These values will
+    /// be ignored when threshold is 1. paymaster, paymasterVerificationGasLimit and paymasterPostOpGasLimit will be
+    /// ignored if paymasterAndData is empty.
+    struct LightUserOpGasLimits {
```

Comment:
vaguely tempted to pack a few of these to save the calldata but probably not worth it idk

Reply:
If packing is added, one suggestion would be to pack the `callGasLimit` and `verificationGasLimit` values into a `bytes32` value called `accountGasLimits`, since that would match the `PackedUserOperation` struct itself

I think it'd also be possible to pack `preVerificationGas` and `maxPriorityFeePerGas` together, but that pairing would be a bit less natural and wouldn't match `PackedUserOperation`